extends Node2D

# Variables para guardar la selecci贸n del jugador
var seleccion = null
var correcto = "izquierda"  # la imagen correcta

func _ready():
	# Conectamos se帽ales de los botones/personajes y el bot贸n rojo
	$Button.connect("pressed", self, "_on_personaje_izq_pressed")
	$Button2.connect("pressed", self, "_on_personaje_abajo_pressed")
	$Button3.connect("pressed", self, "_on_personaje_der_pressed")
	$Button4.connect("pressed", self, "_on_boton_rojo_pressed")


# Cuando se selecciona un personaje
func _on_personaje_izq_pressed():
	seleccion = "izquierda"
	print("Seleccionaste el personaje izquierdo")

func _on_personaje_abajo_pressed():
	seleccion = "abajo"
	print("Seleccionaste el personaje de abajo")

func _on_personaje_der_pressed():
	seleccion = "derecha"
	print("Seleccionaste el personaje derecho")


# Cuando se aprieta el bot贸n rojo
func _on_boton_rojo_pressed():
	if seleccion == null:
		print("Debes seleccionar un sospechoso primero")
		return
	
	if seleccion == correcto:
		mostrar_mensaje("隆Lo lograste! ")
	else:
		mostrar_mensaje("Fallaste, el hacker te descubri贸 ")


# Funci贸n para mostrar un mensaje en pantalla
func mostrar_mensaje(texto: String):
	$Label.text = texto

 
